packages/css-library/src/stylesheets/base/va.scss

```diff
@@ -106,7 +106,7 @@ a {
 }
 
 // ====== Headings
-@import 'headings';
+// @import 'headings';
```

Comment:
I'm temporarily removing this because it contains v3 heading styles which are colliding with Formation heading styles when attempting to swap core imports in vets-website. 

`headings.scss` imports uswds v3 `usa-headings-styles`, which includes some things we may or may not want. For example, [this typeset-heading](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/mixins/typography/typeset.scss#L43-L53) that adds margin top with a universal selector, and uses theme paragraph margin settings. 

I attempted to reconcile these with the [existing heading styles](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/css-library/src/stylesheets/formation-overrides/elements/typography.scss#L80), but felt like I was beginning to change too many things at once. 

We definitely want to get rid of the stuff in the formation-overrides folder, but we need a more systematic approach for this.